x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/api_key/40_view_role_descriptors.yml

```diff
@@ -21,7 +21,7 @@ setup:
               ],
               "applications": [
                 {
-                  "application": "myapp",
+                  "application": "apm",
```

Comment:
just a preference, or did something change ?

Reply:
ah this is necessary since we are now enforcing strict role validation on requests made by all users, including operators. in serverless, only a known list of app names is supported (`apm` being one of them).